Src/ToastNotifications/Display/NotificationsDisplaySupervisor.cs

```diff
@@ -61,6 +61,11 @@ private void Close(INotification notification)
 
         private void InternalClose(INotification notification)
         {
+            //Handle a case where client calls CustomMessage.Close() 
+            //but this object has already been disposed
+            if (_lifetimeSupervisor == null)

```

Comment:
If user closes the notification via close button it disposes NotificationsDisplaySupervisor object.  If we then call CustomMessage.Close() it causes a call to NotificationsDisplaySupervisor.InternalClose() and a crash because _lifetimeSupervisor is null.